pantheon-bundle/frontend/src/app/moduleDisplay.tsx

```diff
@@ -175,11 +175,11 @@ class ModuleDisplay extends Component<any, any, any> {
     private fetchModuleDetails = async(data) => {
         await this.getVariantParam()
         this.setState({
-            modulePath: data.location.pathname,
-            releasePath: "/content" + data.location.pathname + ".preview?variant=" + this.state.variant
+            modulePath: data.location.pathname.substring(7),
+            releasePath: "/content" + data.location.pathname.substring(7) + ".preview?variant=" + this.state.variant
```

Comment:
the value is repeating within the context, can we use it through assigned variable directly ?

Reply:
I updated the code to make it more readable.